src/types.ts

```diff
@@ -65,58 +76,68 @@ export interface PagerViewProps {
   onPageScrollStateChanged?: (event: PageScrollStateChangedNativeEvent) => void;
 
   /**
-   * Determines whether the keyboard gets dismissed in response to a drag.
-   *   - 'none' (the default), drags do not dismiss the keyboard.
-   *   - 'on-drag', the keyboard is dismissed when a drag begins.
+   * Orientation of pager.
    */
-  keyboardDismissMode?: 'none' | 'on-drag';
+  orientation?: Orientation;
 
   /**
-   * Blank space to show between pages. This is only visible while scrolling, pages are still
-   * edge-to-edge.
+   * Determines whether it's possible to overscroll a bit
+   * after reaching end or very beginning of pages.
+   */
+  overdrag?: boolean;
+
+  /**
+   * Blank space to show between pages. This is only visible while scrolling,
+   * pages are still edge-to-edge.
    */
   pageMargin?: number;
 
+  /**
+   * When false, the content does not scroll.
+   * The default value is true.
+   */
+  scrollEnabled?: boolean;
+
+  /**
+   * Shows the dots indicator at the bottom of the view.
+   */
+  showPageIndicator?: boolean;
+
   style?: ReactNative.StyleProp<ReactNative.ViewStyle>;
+
   /**
-   * Set the number of pages that should be retained to either side
-   * of the currently visible page(s). Pages beyond this limit will
-   * be recreated from the adapter when needed.
-   * Defaults to RecyclerView's caching strategy.
-   * The given value must either be larger than 0.
-   *
-   * Only supported on Android.
+   * Transition style of pager.
    */
-  offscreenPageLimit?: number;
+  transitionStyle?: TransitionStyle;
+}
 
-  children: ReactNode;
+export interface PagerViewProps<ItemT> extends EagerPagerProps {
+  /**
+   * Number of pages to render before/after the current page. Minimum 1.
+   */
+  buffer?: number;
```

Comment:
What is a difference between `offscreenPageLimit` and `buffer` ?

Reply:
`offscreenPageLimit` is an Android only concept, of how many (offscreen) fragments are held in native side memory.  iOS does not exactly have an equivalent.  The iOS native side pager (essentially) only ever holds at most three pages: previous (if not at beginning), current, and next (if not at end).

`buffer` controls how many views are rendered js side.  These react views are sent as a batch to the native side, which uses them to populate pages.  The smaller the `buffer`, the more likely a user could swipe to a page that does not yet have its content ready to display; native code will display a placeholder (blank page) until the next batch of rendered views arrives from js side.  This is the work around for native code not being able to synchronously request a specific page be rendered.